test/browser_tests/conf.js

```diff
@@ -1,32 +1,118 @@
 'use strict';
 
-var env = require( './environment.js' );
-
-exports.config = {
-  multiCapabilities: [
-    {
-      browserName: 'chrome',
-      specs:       [ 'spec_suites/shared/*.js' ]
-    },
-    {
-      browserName: 'firefox',
-      specs:       [ 'spec_suites/shared/*.js' ]
-    },
-    // Large Screen only tests
-    {
-      browserName:   'chrome',
-      specs:         [ 'spec_suites/large_screen/*.js' ],
-      chromeOptions: {
-        args: [ '--lang=en',
-                '--window-size=1200,900' ]
+var settings = require( './settings.js' );
+var defaultSuites = require( './default-suites.js' );
+
+function _paramIsSet( param ) {
+  return typeof param !== 'undefined';
+}
+
+/*
+* Params that need to be passed to protractor config.
+*/
+function _retrieveProtractorParams( params ) { // eslint-disable-line complexity, no-inline-comments, max-len
+  var parsedParams = {};
+
+  var checkParam = params.specs;
```

Comment:
Why define this to a var here and below? It's more confusing to follow along with what `checkParam` is set to in each chunk and you're gaining very little since it's only used twice for each param.